src/main/groovy/adamatti/web/RestView.groovy

```diff
@@ -1,7 +1,9 @@
 package adamatti.web
-
+import adamatti.bizo.TiddlerRenderBO
 import adamatti.commons.TemplateHelper
 import adamatti.model.dao.TiddlerDAO
+import adamatti.model.entity.Tiddler
+import com.newrelic.agent.deps.com.google.common.collect.ImmutableMap
```

Comment:
You are right, my bad! 

Will change this, I'm habituated to use ImmutableMap from java8 std lib.